<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DayNightScheEdit</class>
 <widget class="QWidget" name="DayNightScheEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_title">
     <property name="text">
      <string>Schedule Edit</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_background" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="7" column="0">
         <widget class="QLabel" name="label_colorMode">
          <property name="text">
           <string>Color Mode</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_irCut">
          <property name="text">
           <string>IR-CUT</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="ComboBox" name="comboBox_maxShutter"/>
        </item>
        <item row="7" column="1">
         <widget class="ComboBox" name="comboBox_colorMode"/>
        </item>
        <item row="3" column="1">
         <widget class="MyLineEdit" name="lineEdit_gainLevel"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_gainLevel">
          <property name="text">
           <string>Limit Gain Level</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_irLed">
          <property name="text">
           <string>IR LED</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_exposureLevel">
          <property name="text">
           <string>Exposure Level</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="ComboBox" name="comboBox_latemcyIrCut"/>
        </item>
        <item row="1" column="1">
         <widget class="ComboBox" name="comboBox_minShutter"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_maxShutter">
          <property name="text">
           <string>Maximum Shutter</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="ComboBox" name="comboBox_irLed"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_latencyIrCut">
          <property name="text">
           <string>IR-CUT Latency</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_minShutter">
          <property name="text">
           <string>Minimum Shutter</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="ComboBox" name="comboBox_exposureLevel"/>
        </item>
        <item row="5" column="1">
         <widget class="ComboBox" name="comboBox_irCut"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_ok">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_cancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
